### PR TITLE
Shutup presubmit

### PR DESCRIPTION
**Pull request by thockin**
Don't dump stuff all over the screen during presubmit checks.

Make build-go an explicit part of the presubmit, rather than being run a little here and a little there.  sadly, it's just not fast enough.

### CODE REVIEW

File: hack/lib/etcd.sh [hunk 1]
```diff
@@ -40,13 +40,13 @@ kube::etcd::start() {
...
   ETCD_DIR=$(mktemp -d -t test-etcd.XXXXXX)
-  kube::log::usage "etcd -data-dir ${ETCD_DIR} --bind-addr ${host}:${port} >/dev/null 2>/dev/null"
+  kube::log::info "etcd -data-dir ${ETCD_DIR} --bind-addr ${host}:${port} >/dev/null 2>/dev/null"
   etcd -data-dir ${ETCD_DIR} --bind-addr ${host}:${port} >/dev/null 2>/dev/null &
   ETCD_PID=$!
 
   echo "Waiting for etcd to come up."
   kube::util::wait_for_url "http://${host}:${port}/v2/machines" "etcd: " 0.25 80
-  curl -X PUT "http://${host}:${port}/v2/keys/_test"
+  curl -fs -X PUT "http://${host}:${port}/v2/keys/_test"
```
Comment (lavalamp) [inline hack/lib/etcd.sh]: optional: ```--fail --silent``` is easier to read

File: hooks/pre-commit [hunk 2]
```diff
@@ -8,12 +8,22 @@ KUBE_HOOKS_DIR="$(dirname "$(test -L "$0" && echo "$(dirname $0)/$(readlink "$0"
 
 exit_code=0
 
+echo -ne "Checking that it builds... "
+if ! OUT=$("hack/build-go.sh" 2>&1); then
```
Comment (lavalamp) [inline hooks/pre-commit]: Not sure how I feel about the pre-commit hook blowing away the binaries in my directory.

**Comment by lavalamp** [comment]
LGTM, although I'd be more enthusiastic if the build-go just built the necessary things.

**Comment by thockin** [comment]
This did not change anything wrt blowing away binaries - pre-commit already
runs build-go.sh (and still does more than once).  Regarding -fs, I just
followed suit with other places..

I agree that it is horrible that build-go.sh needs THIRTEEN SECONDS to do
nothing.

**Comment by bprashanth** [comment]
This ok to merge right?

**Comment by thockin** [comment]
I hope so.

**Pull request merged**